blitz.py

```diff
@@ -265,7 +271,8 @@ def game_stage():
         current_stage = 30
     elif (
         memory.main.blitz_own_score() - memory.main.blitz_opp_score() >= 1
-        and 570 < memory.main.get_story_progress() < 700
+        and 570 < memory.main.get_story_progress()
```

Comment:
This also looks to be the same. Although, it could use some `()` around things to make sure the `and` is doing what we expect it to be doing. I think without the `()` its fine, but we can enforce that for certain with a few well placed `()`. Also same thing about `get_story_progress()` as a variable.